Packs/Infinipoint/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### Infinipoint 
+- Upgrade of test playbook
```

Comment:
No need to add here the changes in the test playbook
You need to write here only the things that user will see, for example
 - Updated the **infinipoint-run-queries** command name to **infinipoint-execute-action**.